### DISCUSSION TITLE
SQL上线页面无法联想表结构字段，也无法自动补全

### DISCUSSION THREAD

**Discussion by pengkang1991** [hhyo/Archery#2359] [Q&A]
### 重现步骤

在SQL上线页面想修改数据，发现该页面无法联想字段，无法自动补全。但是在SQL查询界面是可以实现的，希望在SQL上线页面也能实现SQL表字段自动补全的功能，但是SQL查询那里的补全也有问题，我要输字段，它给的表名的补全提示，这不对。

### 预期外的结果

<img width="398" alt="企业微信截图_16981127508838" src="https://github.com/hhyo/Archery/assets/49478192/26ae7ce5-4e98-48c3-bb0f-a1228ce34d2d">
<img width="327" alt="企业微信截图_16981127652044" src="https://github.com/hhyo/Archery/assets/49478192/b310e052-44b8-424e-a85e-a37e00ad19bf">
![企业微信截图_16981137378528](https://github.com/hhyo/Archery/assets/49478192/0b79fc04-2761-4b9f-9781-f59e3cde8d8c)

### 版本

1.10.0

### 部署方式

Docker

### 是否还有其他可以辅助定位问题的信息？比如数据库版本等

MySQL: 5.7.26
Browsers: Chrome

**Comment by LeoQuote** [comment]
master 版本试过了吗

**Reply by simplegaoxf** [reply]
作者能帮忙解决一下吗

**Comment by pengkang1991** [comment]
> master 版本试过了吗

我用的是容器镜像方式部署的，1.10.0

**Comment by simplegaoxf** [comment]
我用好几个镜像版本都不行，docker-compose 搭建的，也用了 master，sql 查询以及 sql 上线都没有字段补全。
